### DISCUSSION TITLE
ThinkPad X1 Carbon x Arch Linux

### DISCUSSION THREAD

**Discussion by giscus** [JmPotato/Pomash#18] [Announcements]
# ThinkPad X1 Carbon x Arch Linux

https://ipotato.me/article/75

**Comment by RinChanNOWWW** [comment]
确实折腾啊😂

**Reply by JmPotato** [reply]
你也整一个！

**Reply by RinChanNOWWW** [reply]
还是 Ubuntu 舒服。

**Comment by Zheaoli** [comment]
但是 secure boot 的话，那是每个版本内核都需要自己手动签名一次？

**Reply by JmPotato** [reply]
sbctl 会配置一个 pacman hook 在每次升级内核的时候自动签名，所以不需要每次都手动操作：https://wiki.archlinux.org/title/Unified_Extensible_Firmware_Interface/Secure_Boot#Automatic_signing_with_the_pacman_hook

**Reply by Zheaoli** [reply]
欸，有意思，我去看看

**Comment by xBINj** [comment]
X1 Carbon还可以啊，我18年开始用的一直到现在，不过还是Gen 8。Linux我就想弄个Linux Lite，简单点，反正我也是windows用户，换到Linux也只是玩玩，也不研究啥，简单一点的感觉也还行。

**Comment by waruto210** [comment]
感觉可以考虑买一个X Elite的笔记本折腾一下。

**Reply by RinChanNOWWW** [reply]
怎么突然挖坟?

**Reply by JmPotato** [reply]
很遗憾，X1 已经被我卖掉了，反抗果子暴政失败。
